bitbots_vision/bitbots_vision/vision_modules/yoeo/vision_components.py

```diff
@@ -586,8 +586,8 @@ class DebugImageComponent(IVisionComponent):
     class Colors:
         # BGR
         ball = (0, 255, 0)  # green
-        team_mates = (153, 51, 255)  # magenta
-        opponents = (255, 255, 102)  # cyan
+        team_mates = (255, 0, 0)  # blue
+        opponents = (0, 0, 255)  # red
```

Comment:
Actually, I deliberately chose not to use red and blue because it might be confusing if blue robots get a red bounding box and vice versa. The vision only distinguishes between team mates and opponents, so the bounding box color changes depending on the current game state.

Reply:
Good point

Reply:
@jaagut

Reply:
I see. That's right. I reverted my change.